tools/autograd/derivatives.yaml

```diff
@@ -348,10 +348,10 @@
   self: zeros_like(grad)
 
 - name: norm(Tensor self, Scalar p=2)
-  self: norm_backward(grad, self, p)
+  self: norm_backward(grad, self, p, result)
 
 - name: norm(Tensor self, Scalar p, int64_t dim, bool keepdim=False)
-  self: norm_backward(grad, self, p, dim, keepdim)
+  self: norm_backward(grad, self, p, destination, dim, keepdim)
```

Comment:
Is `result` saved as a `SavedVariable` here, or is it just a scalar?

Reply:
It's saved as a `SavedVariable`